python/CMakeLists.txt

```diff
@@ -417,8 +417,6 @@ if (UNIX)
   set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
 endif()
 
-SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
```

Comment:
Does this really have an effect. Reading the documentation for `CMAKE_BUILD_WITH_INSTALL_RPATH` above it seems like they affect each other a bit.